.github/workflows/sync_docs.yml

```diff
@@ -82,6 +67,29 @@ jobs:
             ${{ runner.os }}-website-
       - run: yarn install --frozen-lockfile
         working-directory: pantsbuild.org
+
+      # Calculate destination
+      - name: Calculate destination
+        id: get-destination-dir
+        run: |
+          BASE_VERSION=$(echo "${{ inputs.version }}" | sed -E 's/^([0-9]+)\.([0-9]+)\.\w+.*/\1.\2/')
+          DESTINATION_DIR="versioned_docs/version-$BASE_VERSION"
+          if [ ! -d "$DESTINATION_DIR" ]; then
+            if [[ "${{ inputs.version }}" =~ .*\.a0$ ]];
+              # a0 => we've cut a new branch from `main`, so make a new version
+              npm run docusaurus docs:version $BASE_VERSION
```

Comment:
is this idempotent (or does it not matter?)

Reply:
It's not, but it creates the `$DESTINATION_DIR`, so it shouldn't re-run as it'll exist (see line 77).

I also suspect the command will error on making a duplicate version, so at least it wouldn't do anything bad.